Decred Wallet/Assets.xcassets/menu/security.imageset/Contents.json

```diff
@@ -6,11 +6,11 @@
     },
     {
       "idiom" : "universal",
-      "filename" : "securitymenuicon.png",
       "scale" : "2x"
     },
     {
       "idiom" : "universal",
+      "filename" : "securitymenuicon3.png",
```

Comment:
Why do we have a `securitymenuicon3`? I see that `securitymenuicon` was deleted. Why not name this new png `securitymenuicon.png` instead of `securitymenuicon3.png`?

Reply:
Renamed